beanstalk-ec2-model-serving/src/main/resources/Images/Smiley_Face.txt

```diff
@@ -0,0 +1 @@
+iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAADGElEQVR4Xu1bv0uyQRz/CKkJCQoaCk6NIUIQSBhIjS2CuOhkNtTgn5BL/g0NRZAuSoODQ2PhYKAIDgquDYEgNihBWmq+nPC8vL2kdw+eFz7ebXrf53vfz+f767zn1AEYY4WHThIgI0CmgKwBK1wDIbQIms1mvL29Ufne3t5Go9GgyvEQEEbAwcEBHh8fodORJWeP8XjMJEfTwzIvjIDDw0M8PDwwAVtpAnZ2dlCtVpmIYvEwTUZYBPh8PhSLRSqwXq+H9fV1qhwNGOu8MAJYU0CzBKytrWEwGFA92+/3YTQaqXKsHqbJCYsAk8mE9/d3KjDNEkC8SsCRSBiNRlMdc3JygpubGypRNM+yzguLAGIQaW+RSATZbHaqfbu7u6hUKtol4Pj4GKlUaioBfr8fhUJhdQlQIoVlx8ga5rPkhKdANBpFOp2eaTtJFbIX+Pj44IFxpg7hBMRiMdze3s40qlarwePxLBw8WUAoAZ+fn7Db7eh2u0LAsSwilAAWg0TLLC0BpE6QSLJYLHNxtpQEGAyGSYE8PT3F9fX17xOQyWQQDoeF9W7i/UkBYzhcobHDJQK8Xi9KpRL0ej2GwyFtzbnmeYLn2gV4G/YTS4lEAhcXF3h+fsbW1tZcRCoPc4kAoszpdKLZbKLVasHhcHAx7l8lwWAQuVwOnU4HVquVm35uBBCLkskkzs/PueWngvLo6Aj39/fc9XJNAcVY5UyPfI7H47i8vJzLWy8vL3C5XAsBvxACFLRfX19/q/TZ2Rmurq6YiQgEAri7u5ucDJHx9PSE/f195ufVCHJNgf8XdrvdqNfr374mxXJzc3PyHdnIkGMyMmw2G9rt9jfZ19fXydZ5kWOhBCiGkzdC5MVIPp+nYiEE7e3toVwuU2V5CAghYJqhoVAIGxsbk1Sh/ULkAfYnHb9KwKJAqdErCZBXZOQVGXlFRl6RUVM1tSYru4DsArILyC4gu4DWKrsaPLILyC4gu4DsArILqKmaWpOVXUB2Aa3FtEo89H8vqFS4bOKSgGXzGG97ZQTwZnTZ9MkIWDaP8bZ35SPgD3PSDxkcJ3pXAAAAAElFTkSuQmCC
```

Comment:
Can you put the images on S3 instead of checking them into git

Reply:
Should I just put this in the public s3 where I think the hugging face model is hosted?